src/compiler.rs

```diff
@@ -247,7 +264,12 @@ pub fn compile_tokens(scope: usize, ctokens: Expression) -> String {
 	while let Some(t) = ctokens.next() {
 		result += &match t {
 			SYMBOL(lexeme) => lexeme,
-			VARIABLE {local, names, values, line} => {
+			VARIABLE {
+				local,
+				names,
+				values,
+				line,
+			} => {
```

Comment:
is this necessary? it looks kinda ugly to me

Reply:
I think that's from the autoformatter

Reply:
I'd understand if it tried to make it not go past screen border, but it didnt go past before so it's weird lol